engineyard/ey-cookbooks-stable-v6
# [FB-835] Set ruby version from EY_RUBY_VERSION on attributes file

#### Description of your patch
Set ruby version from EY_RUBY_VERSION on attributes file
https://t3.fogbugz.com/f/cases/835

#### Recommended Release Notes
Set PATH with the correct Ruby version on cron and unicorn when using EY_RUBY_VERSION.

#### Estimated risk
Low

#### Components involved
Ruby

#### Description of testing done
See QA instructions

#### QA Instructions
Set EY_RUBY_VERSION environment variable to 2.6.1 (or any version that's not our default)
Boot an enviornment using unicorn
Check /data/todo_V6/shared/config/env that it has the correct PATH

```
PATH="/opt/rubies/ruby-2.6.1/bin:/data/todo_V6/current/ey_bundler_binstubs:$PATH"
```
